package.json

```diff
@@ -19,7 +19,7 @@
   ],
   "scripts": {
     "lint": "node scripts/jshint.js",
-    "test": "node tools/test.js",
+    "test": "node --expose-gc tools/test.js",
```

Comment:
To make Browserify happy, another option is to replace this commit with a `complete.exclude("v8");` inside browser_test_generator.js, if that's preferred.